shogun-lib/src/main/java/de/terrestris/shogun/lib/model/User.java

```diff
@@ -54,7 +54,12 @@
 
     @Transient
     @Schema(
-        description = "The user details stored in the associated provider.",
+        description = "The user details stored in the associated provider. Read only. Can only be changed within the provider.",
+        example = "{" +
+            "\"locale\": \"DE\",\n" +
+            "\"lastViewdApp\": \"32\",\n" +
+            "\"lastLogin\": \"2022-12-19\"\n" +
+          "}\n",
```

Comment:
I would also assume the example should be placed inside the field definition as it's generic type here.